Specify the ForeignKeys not as a string

ttung opened the pull request:
### Description
Strings are fickle and can easily be broken.  TableObject.ColumnObject is more durable.

Depends on #77 

### Test plan
Autogenerated a migration and verified that the migration was empty (this is essentially a no-op).

phoenixAja reviewed (approved):
cool this is much nicer

Commit:
Specify the ForeignKeys not as a string

### Description
Strings are fickle and can easily be broken.  TableObject.ColumnObject is more durable.

### Test plan
Autogenerated a migration and verified that the migration was empty (this is essentially a no-op).
